AUTHORS

```diff
@@ -0,0 +1,16 @@
+Corporate Contributors
+======================
+
+Copyright (c) 2015-2017 IBM
+Copyright (c) 2016-2017 Docker, Inc.
+
+Individual Contributors
+=======================
+
+Dan Williams
+Hannes Mehnert
+Ian Campbell
+Madhuri Yechuri
+Martin Lucina
+Ricardo Koller
+Waldir Pimenta
```

Comment:
This lgtm, with the refinement of adding any corporate affiliations after contributor's names in the "Individual Contributors" section above.  My reasoning is that the copyright ownership rests with the corporate contributor in this case, and so it provides a mapping from committer to copyright owner in that case, but still leaves a suitable place for individual contributors to add their name too.